src/declarations/signer/signer.did

```diff
@@ -124,66 +42,14 @@ type SignRequest = record {
   chain_id : nat;
   nonce : nat;
 };
-type Stats = record {
-  user_profile_count : nat64;
-  custom_token_count : nat64;
-  user_timestamps_count : nat64;
-  user_token_count : nat64;
-};
-type SupportedCredential = record {
-  ii_canister_id : principal;
-  issuer_origin : text;
-  issuer_canister_id : principal;
-  ii_origin : text;
-  credential_type : CredentialType;
-};
-type Token = variant { Icrc : IcrcToken };
-type UserCredential = record {
-  issuer : text;
-  verified_date_timestamp : opt nat64;
-  credential_type : CredentialType;
-};
-type UserProfile = record {
-  credentials : vec UserCredential;
-  version : opt nat64;
-  created_timestamp : nat64;
-  updated_timestamp : nat64;
-};
-type UserToken = record {
-  decimals : opt nat8;
-  version : opt nat64;
-  enabled : opt bool;
-  chain_id : nat64;
-  contract_address : text;
-  symbol : opt text;
-};
-type UserTokenId = record { chain_id : nat64; contract_address : text };
 service : (Arg) -> {
-  add_user_credential : (AddUserCredentialRequest) -> (Result);
-  bulk_up : (blob) -> ();
   caller_btc_address : (BitcoinNetwork) -> (text);
   caller_eth_address : () -> (text);
```

Comment:
Why isn't this did file include `caller_btc_balance`? When did you make the release?

Reply:
I saw that the release used is from 2 weeks ago. That answers my question, sorry for the noise!

Reply:
Two weeks ago.  I can make a new release if wanted?

Reply:
Yes, but not for this PR. Let's get this in first.

Reply:
Great thanks!